src/component/reducer/preferences/panelsPreferencesDefaultValues.ts

```diff
@@ -140,11 +140,11 @@ const getPeaksDefaultValues = (
     intensity: { show: true, format: '0.00' },
     fwhm: { show: true, format: '0.00000' },
     mu: { show: false, format: '0.00000' },
+    gamma: { show: false, format: '0.000' },
```

Comment:
How do you decide it has 3 decimals?

Reply:
Ok, then @jobo322 how did you decide? I'm asking since other defaults are either 2 or 5. This is the first occurrence of 3.

Reply:
As I understand it, the format depends on the default values used in the peak shape, so I assume they used this format based on that.